src/Bitmovin/BitmovinClient.php

```diff
@@ -556,6 +557,38 @@ private function createHlsManifestItem($name, EncodingOutput $output)
         return $this->apiClient->manifests()->hls()->create($manifest);
     }
 
+    public function createHlsFmp4Manifest(JobContainer $jobContainer)
```

Comment:
We could rename it to `createHlsFMP4Manifest` as MP4 is written in upper case anywhere else.